.packit.yaml

```diff
@@ -18,7 +18,7 @@ srpm_build_deps:
 actions:
   create-archive:
   - bash -c "sed -i -r \"s/Source0:.+/Source0:\ fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar/\" fido-device-onboard.spec"
-  - bash -c "sed -i \"/Source1/d\" fido-device-onboard.spec"
```

Comment:
I think we still need to keep this for eln builds which are built at part of Fedora

Reply:
This line is kept, if you take a deeper look to the line below the only thing I did is to add an additional rule to remove the comment just above `Source1`.

If you mean that the `Source1` line must be kept in the spec for the ELN builds, I believe it's not going to work anyway: neither the `rust-toolset`  or the `rust-packaging` rpms exist in ELN repos:
```
dnf repoquery --whatprovides rust-packaging --whatprovides rust-toolset
Updating and loading repositories:
Repositories loaded.
```

In CentOS Stream 9:
```
dnf repoquery --whatprovides rust-packaging --whatprovides rust-toolset
Last metadata expiration check: 0:28:25 ago on Fri 01 Mar 2024 10:00:26 AM CET.
cargo-rpm-macros-0:25.2-2.el9.noarch
rust-toolset-0:1.60.0-1.el9.x86_64
rust-toolset-0:1.61.0-1.el9.x86_64
rust-toolset-0:1.62.1-1.el9.x86_64
rust-toolset-0:1.64.0-2.el9.x86_64
rust-toolset-0:1.66.1-1.el9.x86_64
rust-toolset-0:1.69.0-1.el9.noarch
rust-toolset-0:1.71.0-2.el9.noarch
rust-toolset-0:1.71.1-1.el9.noarch
rust-toolset-0:1.73.0-1.el9.noarch
rust-toolset-0:1.75.0-1.el9.noarch
```

In Fedora:
```
dnf repoquery --whatprovides rust-packaging --whatprovides rust-toolset
Last metadata expiration check: 0:31:43 ago on vie 01 mar 2024 09:56:06.
cargo-rpm-macros-0:25.2-1.fc39.noarch
cargo-rpm-macros-0:26.1-1.fc39.noarch
```

Reply:
> If you mean that the `Source1` line must be kept in the spec for the ELN builds, I believe it's not going to work anyway: neither the `rust-toolset` or the `rust-packaging` rpms exist in ELN repos:

It should be fedora builds but the el10 package set, I wonder if there's a gap there because the RHEL package isn't in Fedora.

Reply:
I have done some tests and I am able to build the packages in ELN with `mockbuild` but only using the `koji` mock config.  Regular ELN builds don't work because of the missing package mentioned above. Anyway this discussion is about a change on packit configuration file. In packit we only build fedora and we don't need vendoring, therefore the `Source1` is removed so we don't need to generate/upload the tar file.